gulpfile.js

```diff
@@ -86,7 +86,7 @@ function write(streams, name) {
   return concat(function(body) {
     return file(dest, body, {src: true})
       .pipe(rev())
-      .pipe(gulpif(production, uglify()))
+      .pipe(uglify())
```

Comment:
I don't think we should remove the environment check here. Uglifying this locally makes it impossible to debug.